lib/charms/tempo_k8s/v0/tracing.py

```diff
@@ -93,9 +93,9 @@ def __init__(self, *args):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 7
+LIBPATCH = 8
```

Comment:
As #52 got merged, you'll need to bump up the version by one more.

```suggestion
LIBPATCH = 9
```

Reply:
major bumped instead